src/routes/+page.server.ts

```diff
@@ -0,0 +1,19 @@
+import contentful from "contentful";
+import type { ContentfulClientApi } from "contentful";
+import { CONTENTFUL_SPACE_ID, CONTENTFUL_DELIVERY_API_TOKEN } from "$env/static/private";
+
+// TODO: refactor to a reusable helper function that wraps contenfulClient
```

Comment:
@Benaiah I saw you already had abstracted some of this out in your preview work. I can update to reuse that work here when it's ready